specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/2017-12-01/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I did not see anything about Microsoft.DBforMySQL, will it be included in this PR?

Reply:
i have added mysql.json and examples in this PR.